src/index.ts

```diff
@@ -11,6 +11,7 @@ module.exports = {
     "plugin:prettier/recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "next/core-web-vitals",
+    "prettier",
```

Comment:
we can use types with interfaces now ? don't we need to adjust the `rules --> overrides`  ?https://github.com/supaapps/mysupaapps/pull/14#discussion_r1425321431

Reply:
Yes, extending `prettier` will handle this mission!
Please, test it from your side to make sure of this... 🙏

Reply:
For sure

Reply:
BTW, this will accept using `Type` besides `interface` but with `semi-colons` not `commas`

Reply:
Actually, didn't get this error! 🤔

Reply:
@Moamen-Elgammal if you are testing locally, with `npm link` you will need to run a `npm build` first to make sure you have the latest compiled version in your `eslint-config-supaapps` repo